.snyk

```diff
@@ -13,13 +13,23 @@ ignore:
   SNYK-JS-AGGRIDCOMMUNITY-1932011:
     - '*':
         reason: None Given
-        expires: 2024-07-17T18:27:24.346Z
-        created: 2024-01-18T18:27:24.353Z
+        expires: 2024-03-22T15:43:36.765Z
+        created: 2024-02-21T15:43:36.777Z
   SNYK-JS-AXIOS-6032459:
     - '*':
         reason: Not applicable to axios usage inside node-analytics package
         expires: 2024-10-30T10:18:43.435Z
         created: 2023-10-30T10:18:43.435Z
+  SNYK-JS-INFLIGHT-6095116:
+    - '*':
+        reason: None Given
+        expires: 2024-03-22T15:44:11.367Z
+        created: 2024-02-21T15:44:11.374Z
+  SNYK-JS-IP-6240864:
```

Comment:
`ip` seems to be used by ssh-tunnel:

```
npm ls ip
mongodb-compass-monorepo@ /Users/sergey.petushkov/Projects/MongoDB/compass
├─┬ @mongodb-js/ssh-tunnel@2.1.11 -> ./packages/ssh-tunnel
│ └─┬ socks@2.7.1
│   └── ip@2.0.0
```

are we sure it doesn't affect production?

Reply:
I was reading the error and I thought it was only for testing, but taking a look a second time you are right. Let me upgrade ip in this PR instead of ignoring the warning.

Reply:
(also, yes, `socks` doesn't use the function mentioned in this CVE, so it's fine either way)

Reply:
Just updated socks to 2.7.3 and removed the dependency as you mentioned, now the vulnerability report is green. So I can merge whenever you are ok with it.